### ISSUE TITLE
Community Children's Services

### ISSUE THREAD

**Issue by eatyourpeas** [rcpch/rcpch-audit-engine#198]
Currently there are several children's services missing from the database. If possible, does anyone have a list of the missing commuity services? Our current data structure is:

```
{
        "OrganisationID": "1094758",
        "OrganisationCode": "RJZ70",
        "OrganisationType": "Hospital",
        "SubType": "Hospital",
        "Sector": "NHS Sector",
        "OrganisationStatus": "Visible",
        "IsPimsManaged": "True",
        "OrganisationName": "Orpington Hospital",
        "Address1": "Sevenoaks Road",
        "Address2": "",
        "Address3": "",
        "City": "Orpington",
        "County": "Kent",
        "Postcode": "BR6 9JU",
        "Latitude": "51.362503051757813",
        "Longitude": "9.5164142549037933E-2",
        "ParentODSCode": "RJZ",
        "ParentName": "King's College Hospital NHS Foundation Trust",
        "Phone": "01689 863000",
        "Email": "",
        "Website": "http://pruh.kch.nhs.uk/",
        "Fax": ""
    }
```

**Comment by eatyourpeas** [comment]
In follow up to this issue, as discussed at project board:
In addition to a more complete list of organisations, it would be helpful to have LSOA, MSOA, LAD and CCG codes for each trust. This would allow us to filter hospitals and therefore report at different geographical levels.

**Comment by AmaniKrayemRCPCH** [comment]
Update: We've been unable to find an existing list of community services through the other audits or BAACH. @AmaniKrayemRCPCH is collating a list of community paediatric services from the free-text entries in previous audit submissions. 

Is this data structure up to date? I want to make sure that I include all the relevant details.

**Comment by eatyourpeas** [comment]
The current list is up to date. I am happy to add new organisations.

**Comment by AmaniKrayemRCPCH** [comment]
The NHS trust sites file (ets.csv) on ODS contains community paediatric services. It's worth noting that many community paediatrics services are within hospitals already on the platform, although some aren't. - https://digital.nhs.uk/services/organisation-data-service/export-data-files/csv-downloads/other-nhs-organisations

**Comment by eatyourpeas** [comment]
Thanks I have this list already


From: Amani Krayem ***@***.***>
Date: Friday, 17 March 2023 at 15:56
To: rcpch/rcpch-audit-engine ***@***.***>
Cc: Simon Chapman ***@***.***>, Author ***@***.***>
Subject: Re: [rcpch/rcpch-audit-engine] Community Children's Services (Issue #198)

The NHS trust sites file (ets.csv) on ODS contains community paediatric services. It's worth noting that many community paediatrics services are within hospitals already on the platform, although some aren't. - https://digital.nhs.uk/services/organisation-data-service/export-data-files/csv-downloads/other-nhs-organisations

**Comment by eatyourpeas** [comment]
Epilepsy12 meeting
- [ ] E12 team to curate a list of community services by polling their current audit leads
- [ ] refactor the term HospitalTrust to Organisation, ParentTrust to Trust

**Comment by AmaniKrayemRCPCH** [comment]
@eatyourpeas 
Here you go! [Organisation List - Final.csv](https://github.com/rcpch/rcpch-audit-engine/files/11274971/Organisation.List.-.Final.csv)
This is from trawling through our existing data - specifically free-text responses to the question "Name of referral hospital/community service". I got all of the organisations from [ODS (ets.csv)](https://digital.nhs.uk/services/organisation-data-service/export-data-files/csv-downloads/other-nhs-organisations) , and then referenced [the website datasets](https://www.nhs.uk/about-us/nhs-website-datasets/) to fill in the following columns:
- OrganisationID
- OrganisationType
- OrganisationSubType
- Sector
- OrganisationStatus
- IsPimsManaged
- Latitude
- Longitude
- Phone
- Email
- Website 
- Fax

Many fields are blank because some of the organisations listed in ODS were not in the website dataset files. I'm hoping you have a better source to reference. 

I'm sure there's an easier way than what I've done haha.

**Comment by eatyourpeas** [comment]
This is very helpful thank you
It will take a while to embed this as we now have a database in live dependent on the old organisations (many of which will of course be the same)

S

From: Amani Krayem ***@***.***>
Date: Wednesday, 19 April 2023 at 16:59
To: rcpch/rcpch-audit-engine ***@***.***>
Cc: Simon Chapman ***@***.***>, Mention ***@***.***>
Subject: Re: [rcpch/rcpch-audit-engine] Community Children's Services (Issue #198)

@eatyourpeas<https://github.com/eatyourpeas>
Here you go! Organisation List - Final.csv<https://github.com/rcpch/rcpch-audit-engine/files/11274971/Organisation.List.-.Final.csv>
This is from trawling through our existing data - specifically free-text responses to the question "Name of referral hospital/community service". I got all of the organisations from ODS (ets.csv)<https://digital.nhs.uk/services/organisation-data-service/export-data-files/csv-downloads/other-nhs-organisations> , and then referenced the website datasets<https://www.nhs.uk/about-us/nhs-website-datasets/> to fill in the following columns:

  *   OrganisationID
  *   OrganisationType
  *   OrganisationSubType
  *   Sector
  *   OrganisationStatus
  *   IsPimsManaged
  *   Latitude
  *   Longitude
  *   Phone
  *   Email
  *   Website
  *   Fax

Many fields are blank because some of the organisations listed in ODS were not in the website dataset files. I'm hoping you have a better source to reference.

I'm sure there's an easier way than what I've done haha.

**Comment by AmaniKrayemRCPCH** [comment]
@eatyourpeas 

What's the easiest way to add a new organisation for me and Niky? There are probably some organisations that aren't on the list and, in the future, hospitals that don't participate in the audit may choose to participate. 

Should we just add an issue with the ODS code?

**Comment by eatyourpeas** [comment]
Please
I think that would be the easiest
We have rebuilt with the new organisations, hopefully in live by the end of the day.

For future organisations I agree post an issue with the deets and we will include
S

From: Amani Krayem ***@***.***>
Date: Tuesday, 25 April 2023 at 13:11
To: rcpch/rcpch-audit-engine ***@***.***>
Cc: Simon Chapman ***@***.***>, Mention ***@***.***>
Subject: Re: [rcpch/rcpch-audit-engine] Community Children's Services (Issue #198)

@eatyourpeas<https://github.com/eatyourpeas>

What's the easiest way to add a new organisation for me and Niky? There are probably some organisations that aren't on the list and, in the future, hospitals that don't participate in the audit may choose to participate.

Should we just add an issue with the ODS code?

**Comment by eatyourpeas** [comment]
fixed with #447
